src/Okta.Sdk/Generated/IAuthorizationServerPolicyRule.Generated.cs

```diff
@@ -36,10 +36,10 @@ public partial interface IAuthorizationServerPolicyRule : IResource
         string Type { get; set; }
 
         Task ActivateAsync(
-            string policyId, CancellationToken cancellationToken = default(CancellationToken));
+            string authServerId, string policyId, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
It seems this was wrong before, that's why we're introducing this breaking change?
TO DO: Double check